src/dialog-service.ts

```diff
@@ -185,7 +186,7 @@ export class DialogService {
         }
         return Promise.reject(reason);
       });
-    })).then((unclosedControllers) => unclosedControllers.filter(unclosed => !!unclosed));
+    })).then(unclosedControllers => unclosedControllers.filter(unclosed => !!unclosed));
```

Comment:
Sorry I missed this before but what is this actually filtering?  Seems to be checking that an item in an iterable array is not null, correct?